### DISCUSSION TITLE
Attributed Graph Clustering: A Deep Attentional Embedding Approach

### DISCUSSION THREAD

**Discussion by tb1over** [pyg-team/pytorch_geometric#4252] [Q&A]
hello, is anyone implemented this model
https://arxiv.org/pdf/1906.06532.pdf

**Comment by rusty1s** [comment]
We do not have an example of this in PyG, I am sorry. We can definitely try to integrate it if you are interested in contributing :)

**Reply by tb1over** [reply]
i want to implement this model, but lack of skills, may be a lot of time

**Reply by rusty1s** [reply]
No worries. Feel free to ask anything. Otherwise, I can try to give it a shot as well (just not sure when I will have time to do this:))
